assets/js/components/GoogleChart/index.js

```diff
@@ -287,6 +376,35 @@ export default function GoogleChart( props ) {
 				{ gatheringData && isChartLoaded && (
 					<GatheringDataNotice style={ NOTICE_STYLE.OVERLAY } />
 				) }
+				{ !! dateMarkers.length &&
```

Comment:
Nitpick to say, technically we don't need this check for `dateMarkers.length` as React will simply render nothing for an empty array. However, I'm happy keeping it in to be explicit, too.

Reply:
No worries, let's leave as-is then :)